inc/functions_modcp.php

```diff
@@ -181,11 +181,36 @@ function send_report($report)
 		}
 	}
 
-	while($mod = $db->fetch_array($query))
+	$lang_string_subject = "emailsubject_report{$report_type}";
+	$lang_string_message = "email_report{$report_type}";
+
+	if(!$lang_string_subject || !$lang_string_message)
```

Comment:
Oops.

> add a hook to that function

Forgot to add variables to global scope. You can hook at `report_do_report_start`.